modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/DLCopyValidationUtil.java

```diff
@@ -16,13 +16,22 @@ public class DLCopyValidationUtil {
 
 	public static String getCopyToValidationMessage(
 		long companyMaxSizeToCopy, long groupMaxSizeToCopy,
-		long systemMaxSizeToCopy, long size, Locale locale) {
+		long systemMaxSizeToCopy, long size, Locale locale,
+		boolean checkInDestination) {
```

Comment:
This reads confusing, as it has nothing to do with the "check in" operation. Maybe something like `targetGroupValidationFailed` could work?

Reply:
I renamed it to `targetGroupValidation`